### PR TITLE
HTML Meta Tag

### LINKED ISSUE #6052: Frontend_Web_Development_HTML_CSS_JS : HTML meta tag
### Description

Documentation on HTML meta tag

### Domain

Frontend Dev HTML/CSS/JS

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by palak1101**
<hr>

## Description 📜

- What is HTML Meta tag?
- Key points on meta tag
- Advantages of using meta tag
- Attributes
- Example with output

Fixes #6052

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Web_Development/FrontEnd/HTML_Meta_Tag/MetaTag.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# HTML : Meta Tag 
+
+## What is HTML ```<meta>``` tag ?
```
Comment (2024-SANDHYA) [inline Web_Development/FrontEnd/HTML_Meta_Tag/MetaTag.md:3]: Remove the second sub-heading.

File: Web_Development/FrontEnd/HTML_Meta_Tag/MetaTag.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+The meta-data means a set of data that describes or gives more information about other data of webpage. The ```<meta>``` tag in HTML provides important information about created HTML document, which is relevant to browser. The ```<meta>``` tag is used to define or represent the metadata of the HTML document on browser. It specifies properties of HTML document, such as description of the page, keywords, language, author of the documents, expiry date , viewport setting, etc.
+
+Syntax 
+
+ ```
+ <meta attribute-name="value">
+ ```
```
Comment (2024-SANDHYA) [inline Web_Development/FrontEnd/HTML_Meta_Tag/MetaTag.md:10]: Don't use soo much of  **```** meta **````** everywhere. It's blocking the content.

**Review by 2024-SANDHYA — CHANGES_REQUESTED** [review]
Add image credits too along with other changes & once done ping me up here or on Discord. Closing PR till then you can reopen once you are done with the changes.

File: Web_Development/FrontEnd/HTML_Meta_Tag/MetaTag.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+## Visiblity of metadata on browsers
+
+![metadata](https://user-images.githubusercontent.com/75716335/139000776-830f91e1-fad9-4db0-82e1-3f2fc43525ad.png)
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML_Meta_Tag/MetaTag.md:14]: Add reference website.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
You need not create a new PR for changes, commit in the same.

**Comment by palak1101** [comment]
> Add image credits too along with other changes & once done ping me up here or on Discord. Closing PR till then you can reopen once you are done with the changes.

@2024-SANDHYA I've made the changes you suggested , but the commit is not visible can you open it and see?

**Comment by palak1101** [comment]
> Add image credits too along with other changes & once done ping me up here or on Discord. Closing PR till then you can reopen once you are done with the changes.

@2024-SANDHYA I've made the changes you suggested, but the commit is not visible to me , can you open PR and check ?
as I cannot find option to open PR.

**Review by 2024-SANDHYA — APPROVED** [review]
Perfect !!

**Pull request merged**